definitions/tools/cnvkit_batch.cwl

```diff
@@ -13,64 +13,89 @@ requirements:
       ramMin: 4000
       tmpdirMin: 10000
     - class: InlineJavascriptRequirement
+arguments: [{ valueFrom: "$((inputs.reference.hasOwnProperty('cnn_file'))? '' : '--normal')" }]
```

Comment:
This seems like it might yield an extra positional argument that is the empty string.  Maybe that's harmless.  If it is problematic, you could possibly convert this to a boolean with a prefix.  Then it should follow the normal CWL rules of only including the prefix on `true` values.